.github/workflows/deploy-docker-images.yml

```diff
@@ -4,78 +4,52 @@ name: Push docker images to GitHub container registry
 
 on:
   push:
-    branches: [main, staging]
-    tags: ["v*.*.*"]
-  pull_request:
-    branches: [main, staging]
```

Comment:
Do you think we can reintegrate `pull_requests`? It's pretty nice to test if it breaks with changed on the base branch (if the source branch is not up to date).

Reply:
I think too it needs restoring. I doubt "push" will trigger a build from an external contributor.